core/commands/log_graph.py

```diff
@@ -313,10 +313,15 @@ def on_post_window_command(self, window, command_name, args):
 
 def colorize_dots(view):
     # type: (sublime.View) -> None
-    dots = tuple(_find_dots(view))
+    dots = tuple(find_dots(view))
     _colorize_dots(view.id(), dots)
 
 
+def find_dots(view):
+    # type: (sublime.View) -> Set[colorizer.Char]
+    return set(_find_dots(view))
```

Comment:
When does this happen? multiple cursons on one line?

Reply:
Yes.